modules/teamSearch/src/main/TeamSearchApi.scala

```diff
@@ -1,45 +1,14 @@
 package lila.teamSearch
 
-import akka.stream.scaladsl.*
-
-import lila.core.team.TeamData
 import lila.search.*
 import lila.search.client.SearchClient
-import lila.search.spec.{ Query, TeamSource }
+import lila.search.spec.Query
 
-final class TeamSearchApi(
-    client: SearchClient,
-    teamApi: lila.core.team.TeamApi
-)(using Executor, akka.stream.Materializer)
-    extends SearchReadApi[TeamId, Query.Team]:
+final class TeamSearchApi(client: SearchClient)(using Executor) extends SearchReadApi[TeamId, Query.Team]:
 
   def search(query: Query.Team, from: From, size: Size) =
     client
       .search(query, from, size)
-      .map: res =>
-        res.hitIds.map(TeamId.apply)
+      .map(_.hitIds.map(TeamId.apply))
 
   def count(query: Query.Team) = client.count(query).dmap(_.count)
-
-  def store(team: TeamData) = client.storeTeam(team.id.value, toDoc(team))
-
-  private def toDoc(team: TeamData) =
-    TeamSource(
-      name = team.name,
-      description = team.description.value.take(10000),
```

Comment:
this is fine because We already limit the description: https://github.com/lenguyenthanh/lila/blob/lila-search-team-ingestor/modules/team/src/main/TeamForm.scala